tests/jeos/image_info.pm

```diff
@@ -0,0 +1,127 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: This module is to collect information about JeOS/Minimal-VM
+#          information, like image size, directory sizes, package list,
+#          etc. and push it to an InfluxDB to be displayed in Grafana.
+#          This module doesn't do any interaction with the booted system.
+#          All the operations are done on worker level.
+#          This module is not intended to be a limiting factor to fail a
+#          job, specially pushing data to the DB, therefore no `die`
+#          operations are used.
+# Maintainer: QA-C <qa-c@suse.de>
+
+use Mojo::Base qw(opensusebasetest);
+use testapi;
+use version_utils qw(is_sle is_opensuse is_openstack);
+use mmapi qw(get_current_job_id);
+use Cwd;
+
+sub ls {
+    my $output = `ls -l`;
+    record_info('ls', $output);
+}
+
+sub run {
+    my $self = shift;
+    my $download_url;
+    my $hdd;
+    my $image;
+    my $size;
+
+    my $dir = getcwd;
+    record_info('pwd', $dir);    # For debug purposes
+
+    ls;    # debug
+    if (is_openstack) {
+        # In OpenStack JeOS tests, we boot a different HDD which serves as
+        # a jumphost with the needed CLI tools to upload the image and create
+        # the VM in the remote openStack environment.
+        # The JeOS HDD is not copied in the openQA pool directory, we need to
+        # download using PUBLIC_CLOUD_IMAGE_LOCATION variable
+        my $img_url = get_required_var('PUBLIC_CLOUD_IMAGE_LOCATION');
+        record_info('URL', "Downloading $img_url ...");
+        system("curl -O $img_url");
```

Comment:
Will the downloaded file be removed after the test exits?

Reply:
This is downloaded in the pool directly, so it will be removed entirely after test is finished, but I could do it explicitly here.